### PR TITLE
[ch553] Docker java11

### PR DESCRIPTION
**Pull request by jespada**
- This is the first iteration to try to run  graphq on java11
- It also prepares Jenkinsfile to be able to run linting, static analysis and integration test.

### CODE REVIEW

**Comment by jespada** [comment]
[ch 553]

**Comment by jespada** [comment]
> Dockerfile.jlink could be Dockerfile.

@greut I'll add later on another PR with more doc and k8s part a multi-stage Dockerfile like [this](https://gist.github.com/jespada/e7c9234ee36e8ec8484e8abb15711dac) in case someone wants to build it locally and all the instructions

**Comment by greut** [comment]
for Obwald, `make docker11` runs `lein jlink assemble` locally.

**Comment by jespada** [comment]
@greut thanks for the jdk.crypto

File: Jenkinsfile [hunk 1]
```diff
@@ -25,10 +50,71 @@ node {
   }
 }
 
+def cljLint() {
+  withCredentials([usernamePassword(credentialsId: 'artifact-repository-user', usernameVariable: 'ARTIFACT_USER', passwordVariable: 'ARTIFACT_PASSWORD')]) {
+    docker.withRegistry('https://registry.internal.exoscale.ch') {
+      def clojure = docker.image('registry.internal.exoscale.ch/exoscale/clojure:cosmic')
+      clojure.pull()
+            clojure.inside('-u root --net=host -v /home/exec/.ccm/repository:/root/.ccm/repository -v /home/exec/.m2/repository:/root/.m2/repository -e ARTIFACT_USER -e ARTIFACT_PASSWORD') {
+        // will make eastwood, kbit and joker happy later
```
Comment (alexanderjamesking) [inline Jenkinsfile]: in this PR or a separate PR?

**Comment by greut** [comment]
:warning: `[spootnik/net "0.3.3-beta37"]` might blow up https://github.com/pyr/net/pull/53

File: Jenkinsfile (hunk 1 above)
Comment (jespada) [reply Jenkinsfile]: @alexanderjamesking in another :)

**Comment by jespada** [comment]
> ⚠️ `[spootnik/net "0.3.3-beta37"]` might blow up [pyr/net#53](https://github.com/pyr/net/pull/53)

It doesn't , yet!, its running on prod already

File: Jenkinsfile (hunk 1 above)
Comment (jespada) [reply Jenkinsfile]: in a separate PR

**Comment by greut** [comment]
> its running on prod already

:confused:

**Comment by jespada** [comment]
> > it's running on prod already
> 
> 😕
@greut 
We have no other way to test easily, I did test locally spinning up kafka, graphite, zookeeper etc but that's as far as I can go in my laptop, so got a green light to test in prod (we only have 1 instance of this running there), since the impact is low if it doesn't work, it was safe to perform and easy. Rollback is also easy, hence the decision to test it there

**Pull request merged**